content/02.introduction.md

```diff
@@ -3,25 +3,23 @@
-Furthermore, as pediatric tumors are rare, incidence of any given brain tumor molecular subtype is relatively low.
+Thus, combined with rare incidences of pediatric tumors in the first place, limited availability of tissue for developing patient-derived cell lines and mouse models has hindered research.
 
-To address these long-standing barriers, multiple national and international consortia have collaborated to uniformly collect clinically-annotated surgical biosamples and associated germline materials through both observational and interventional clinical trials.  
+To address these barriers, multiple national and international consortia have collaborated to uniformly collect clinically-annotated surgical biosamples and associated germline materials through both observational and interventional clinical trials.
 The Pediatric Brain Tumor Atlas (PBTA) initiative established in 2018 by the Children's Brain Tumor Network (CBTN, cbtn.org)[@doi:10.1016/j.neo.2022.100846] and the Pacific Pediatric Neuro-Oncology Consortium (PNOC, pnoc.us) built upon 12 years of enrollment, sample collection, and clinical followup across over 30 institutions.
-Just as cooperation accelerates specimens and data sharing, collaboration among researchers including computational and bench scientists, clinicians, and pathologists is critical for rigorous genomic analysis.
+Just as cooperation accelerates specimens and data sharing, collaboration among both computational researchers and bench scientists, clinicians, and pathologists is critical for rigorous genomic analysis.
 
 Although there has been significant progress elucidating genomic bases of pediatric brain tumor formation and progression, translating therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace.
-Within the last 20 years, the FDA has approved only seven targeted agents for treating pediatric brain tumors [@url:https://www.fda.gov/about-fda/oncology-center-excellence/pediatric-oncology-drug-approvals].
-This is partly due to pharmaceutical company priorities, posing challenges for researchers to obtain therapeutic agents for pediatric clinical trials.
-Critically, since August 2020, an amendment to the Pediatric Research Equity Act called the "Research to Accelerate Cures and Equity (RACE) for Children Act" mandates that all new adult oncology drugs also be tested in children when the molecular target exists in a childhood cancer.
-The RACE Act, coupled with genomics advances to identify putative molecular targets in pediatric cancers, will accelerate identification of previously-overlooked but effective therapeutic options for pediatric diseases.
+Within the last 20 years, the FDA has approved only seven targeted agents for treating pediatric brain tumors [@url:https://www.fda.gov/about-fda/oncology-center-excellence/pediatric-oncology-drug-approvals], making it challenging for researchers to obtain therapeutic agents for pediatric clinical trials.
```

Comment:
I do not think these sentences can be combined this way: I believe the implied causality goes the other way. No access means no trials means no approvals.

Reply:
Agree with @jashapiro and in the last PR with @jaclyn-taroni we trimmed these two sentences significantly.